core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -3000,11 +3001,27 @@ private List<String> doGetUserList(String claim, String claimValue, String profi
                 // Get the user list and return with domain appended.
                 try {
                     AbstractUserStoreManager userStoreManager = (AbstractUserStoreManager) userManager;
+
+                    if(userStoreManager.isCircuitBreakerOpen()) {
```

Comment:
```suggestion
                    if (userStoreManager.isCircuitBreakerOpen()) {
```
check formatting in other places as well